example/webpack1-babel/webpack.conf.js

```diff
@@ -18,25 +18,17 @@ module.exports = {
     main: './test/main_test',
   },
 
-  plugins: [
-    new webpack.DefinePlugin({ __VALUEA__: 10 }),
-  ],
+  plugins: [new webpack.DefinePlugin({__VALUEA__: 10})],
```

Comment:
sorry for this noise; I'm guessing this was prettier kicking in since I changed this file?